webtau-feature-testing/examples/scenarios/concept/metaDataMethodBased.groovy

```diff
@@ -0,0 +1,15 @@
+package scenarios.concept
+
+import static com.twosigma.webtau.WebTauGroovyDsl.*
+
+Support.teamA()
```

Comment:
I think I'd still be quite interested in syntax that looks more similar to:
```
metadata {
  team = "foo",
  owner = "blah",
...
}
```
Is that possible with the current API?  Main reason why I like this syntax is it allows you to group and scope all metadata specification together.  Otherwise you'll end up with lots of:
```
Support.teamA()
Support.foo()
Support.blah()
```
This gets a little tedious and worse, someone might end up scattering it all over the file.

Reply:
Hmm, though I suppose that makes the scenario level overriding a little more difficult.

Reply:
It is possible to have different syntaxes. As in the end it boils down to calling a method one can come up with various shortcuts or DSLs. 
If we feel like we need to support metadata closure think that it would be better if we support meta closure out of the box I will add it to the available methods. 

In case you described above test writers can opt for a shortcut like ‘Support.meta(k: v, a: 10)’. 

I wanted to have a syntax that can support strongly defined keys and values.

Reply:
Curious to hear @tsiq-clemens' thoughts on this too as he's the one who actually has a use case for this :)

Reply:
I can see scenario-level overrides in the shape of an optional 2nd parameter (map), e.g.
`scenario('some scenario name', [ owner: 'scenario owner' ]) { ... }`
The implementation would effectively walk through the map calling `attachTestMetaValue(key, 

I am not sure that that would look consistent with the method-syntax to the end user / developer, though.

Reply:
I quite like the declarative nature of the closure syntax; if it's easy to support it has my vote. 
If it's a lot of work, you can get a bit more declarative with the method calls as well, e.g. `Support.supportedByTeam("Billing")` or even more strongly defined `Support.supportedByBilling()`.
There is a danger of scattering those function calls, of course; similarly, you could have multiple metadata closures and mess up the code. At some point, we need to appeal to the Clean Coder in the test developer to keep things organized.

Reply:
With the closure you at least have the option of throwing on the second invocation

Reply:
Can you elaborate on second invocation? Let’s move it back to the issue where we discussed the syntax. I may not fully understood the problem.